hw/ip/kmac/dv/kmac_base_sim_cfg.hjson

```diff
@@ -2,10 +2,7 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 {
-  // Name of the sim cfg - typically same as the name of the DUT.
-  name: kmac
-
-  // Top level dut name (sv module).
+  // Top level dut name
```

Comment:
Probably clarify with a comment stating that this is expected to be set by the HJson that imports this file.

Reply:
Done.